cwltool/load_tool.py

```diff
@@ -165,8 +165,8 @@ def validate_document(document_loader,  # type: Loader
             r"^(?:cwl:|https://w3id.org/cwl/cwl#)", "",
             workflowobj["cwlVersion"])
     else:
-        _logger.warning("No cwlVersion found, treating this file as draft-2.")
-        workflowobj["cwlVersion"] = "draft-2"
+        _logger.warning("No cwlVersion found, treating this file as v1.0")
```

Comment:
Yes, instead of falling back it should be a fatal error.  We don't want to encourage people to start writing files cwl where we can't identify the version, it makes spec evolution much harder.